process/transaction/shardProcess.go

```diff
@@ -1187,6 +1208,36 @@ func isNonExecutableError(executionErr error) bool {
 		errors.Is(executionErr, process.ErrTransactionNotExecutable)
 }
 
+func (txProc *txProcessor) createCompleteEventLog(scr data.TransactionHandler, originalTxHash []byte) {
+	scrHash, err := core.CalculateHash(txProc.marshalizer, txProc.hasher, scr)
+	if err != nil {
+		scrHash = originalTxHash
```

Comment:
do we need a log message for this error?

Reply:
not needed